flexmeasures/data/models/migration_utils.py

```diff
@@ -1,3 +1,8 @@
+"""
+This module is part of our data model migration (see https://github.com/SeitaBV/flexmeasures/projects/9).
```

Comment:
It depends on which modules you want to apply it to. Two thoughts that come to mind:
- Renaming is nice because it let's you know when you're about to use legacy code while coding.
- Renaming an entire module as part of a PR that is not entirely dedicated to renaming modules might make the reviewing process much harder, because it's not easy to distinguish in-module changes from module renaming. However, this argument doesn't apply to entirely new modules, like the present `migration_utils.py`.

Shall I rename this module to `_legacy_migration_utils.py`?

Reply:
Yes in your case it will look okay. I'm not sure if an underscore in the beginning works well, I'd just use "legacy".

My PR replaces some files, and thus might create some clutter. I'll have to see about that.